Use phantom ink instead of managlass to make abstruse platforms invisible

zacharybarbanell opened the pull request:
#4279

TheRealWormbo committed:
Add toggle option to Manufactory Halo (implements #4277)

zacharybarbanell committed:
Phantom ink is now used instead of Managlass to completely hide abstruse and spectral platforms

zacharybarbanell committed:
update changelog.md

zacharybarbanell committed:
inking platforms no longer consumes the ink

zacharybarbanell committed:
Merge branch '1.19.x' into 4279

zacharybarbanell committed:
removed unused imports

zacharybarbanell committed:
Merge branch '4279' of https://github.com/zacharybarbanell/Botania into 4279

zacharybarbanell committed:
autoformatters

TheRealWormbo committed:
Fixed Teru Teru Bozu happy face rotation

TheRealWormbo committed:
Raised Teru Teru Bozu display in 1st person hand to make sad face visible

williewillus committed:
Adjust lexicon text now that Rosa Arcana's XP pickup yield matches its player-based yield

zacharybarbanell committed:
Added a tag for items used to combine lenses, and added honey bottles to that tag

williewillus commented on Xplat/src/main/java/vazkii/botania/common/block/block_entity/PlatformBlockEntity.java:
```diff
@@ -107,9 +107,6 @@ public void writePacketNBT(CompoundTag cmp) {
 		BlockState state = NbtUtils.readBlockState(cmp.getCompound(TAG_CAMO));
-		if (state.isAir()) {
-			state = null;
-		}
```
I'm very slightly concerned what happens if the camo state is set to a block that is later removed from the game (e.g. a modded block), because then the camoState would turn into air (whereas before this PR it would turn back into an uncamouflaged platform). Not sure if it's a problem.

williewillus committed:
Fix Rod name typo in 1.16.1-403 changelog

Dawnwalker666 committed:
Upload zh_cn files (#4366)

The main work is to sort the entries, add missing entries and fix some
mistranslations.
In addition, it makes the number of zh_cn lines aligned with en_us,
which is convenient for future Chinese translation. Currently in this
file, the number of zh_cn lines = en_us + 2
The reason for this is that Chinese files have two more lines of opening
comments than English.

williewillus committed:
Fix spotless

Dawnwalker666 committed:
Upload zh_cn files (#4375)

zacharybarbanell committed:
Add code formatting to a command in CONTRIBUTING.md which didn't have it

zacharybarbanell committed:
Make dispensers able to use pasture seeds (#4360)

Closes #4344

zacharybarbanell commented:
Sounds good to me.

On Fri, May 3, 2024 at 4:48 PM Wormbo ***@***.***> wrote:

> I'm going to assume nothing is happening with this PR anymore. I'll take
> the changes and finalize them in a new PR targeting the 1.20 branch.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/VazkiiMods/Botania/pull/4362#issuecomment-2093805680>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5VZX62TDK2ZWAAEZJJ6OLZAQAZZAVCNFSM6AAAAAAYAFNZL2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJTHAYDKNRYGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
